app/src/main/java/com/sun/heartrate/data/source/HeartLocalDataSource.kt

```diff
@@ -0,0 +1,30 @@
+package com.sun.heartrate.data.source
+
+import com.sun.heartrate.data.database.HeartDatabase
+import com.sun.heartrate.data.model.HeartModel
+import com.sun.heartrate.data.model.OnDataLoadedCallback
+
+class HeartLocalDataSource(
+    private val heartDatabase: HeartDatabase
+) : HeartDataSource {
+    override fun insetHeart(heartModel: HeartModel, onDataLoadedCallback: OnDataLoadedCallback<Boolean>) {
+        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
```

Comment:
Nên xoá mấy cái to do này đi.

Reply:
chưa implement thì nên để TODO chứ em

Reply:
Thế ạ. Giờ em mới biết. :))